src/src/com/microsoft/aad/adal/AuthenticationConstants.java

```diff
@@ -349,4 +349,12 @@
     public static final int DEFAULT_EXPIRATION_TIME_SEC = 3600;
 
     public static final String AUTHENTICATION_FILE_DIRECTORY = "com.microsoft.aad.adal.authentication";
+    
+    protected static final class OAuth2ErrorCode {
```

Comment:
my thought is, in case we are adding more error code in the future, group oauth error code constants to the internal class. I don't want the class to be visible outside which is only for internal use.